account/bitcoin/privkey.go

```diff
@@ -0,0 +1,37 @@
+package bitcoin
+
+import (
+	"github.com/DE-labtory/zulu/account/bitcoin/chaincfg"
+	"github.com/DE-labtory/zulu/keychain"
+	"github.com/DE-labtory/zulu/types"
+	"github.com/btcsuite/btcutil"
+	"github.com/btcsuite/btcutil/hdkeychain"
+)
+
+type PrivKey struct {
+	*hdkeychain.ExtendedKey
+}
+
+func GetPrivKey(key keychain.Key) (*PrivKey, error) {
+	extKey, err := hdkeychain.NewKeyFromString(string(key.PrivateKey))
```

Comment:
```suggestion
	extKey, err := hdkeychain.NewKeyFromString(hex.EncodeToString(key.PrivateKey))
```
I recommend to use the `encoding/hex` package to properly convert hexadecimal representation of byte to hex string (see example: https://play.golang.org/p/Yp0y1zOrPe1).